forge-gui/res/cardsfolder/upcoming/lithomantic_barrage.txt

```diff
@@ -0,0 +1,12 @@
+Name:Lithomantic Barrage
+ManaCost:R
+Types:Sorcery
+K:This spell can't be countered.
+A:SP$ Pump | ValidTgts$ Creature | IsCurse$ True | AIValid$ Creature.White,Creature.Blue,Planeswalker.White,Planeswalker.Blue | RememberObjects$ Targeted | SubAbility$ DBDmg | SpellDescription$ CARDNAME deals 1 damage to target creature or planeswalker. It deals 5 damage instead if that target is white and/or blue.
```

Comment:
AITgts$ is the param you want

Reply:
Also for Lightning Dart in this PR?

Reply:
You also need to:
 - allow Planeswalkers in ValidTgts
 - add TgtPrompts

Reply:
StackDescription$ REP target creature or planeswalker_{c:Targeted}

Then DBDmg line may need `StackDescription$ None`